tests/wpunit/Tribe/Events/ORM/Events/CreateTest.php

```diff
@@ -16,6 +16,14 @@ public function setUp() {
 		$this->factory()->organizer = new Organizer();
 		// To support taxonomy term creation and assignment.
 		wp_set_current_user( $this->factory()->user->create( [ 'role' => 'administrator' ] ) );
+
+		if ( isset( tribe( 'cache' )['option_gmt_offset'] ) ) {
```

Comment:
We do this _so.many.times_ - can we use a function or filter for this? (not pressing)

Reply:
That is an amazing suggestion i added `tribe( 'cache' )->reset();`